### ISSUE TITLE
Does it support the rule index's name like regex result $1 $2

### ISSUE THREAD

**Issue by fly-studio** [siddontang/go-mysql-elasticsearch#268]
First I had many databases to sync, and ES 6 has no type,  So I add a prefix to every index name by table_name

I'll set Each tables like this
```
[[rule]]
schema = "db1"
table = "table1"
index = "db1-table1"
type = "_doc"
```

Does this support a wildcard result?

```
[[rule]]
schema = "db1"
table = "(.*)"
index ="db1-$1"
type = "_doc"
```

**Comment by siddontang** [comment]
em, seem we don't support wildcard like this now. but maybe we can do it, refactoring rule, for every table, get associated index and type dynamically.

**Comment by 10000go** [comment]
@fly-studio Did you solve the rule wildcard problem later?
I also need to prefix all index like this:
mysql table: table_1 table_2
db1 to es index:db1_table_1 db1_table2
db2 to es index:db2_table_1 db2_table2
thx!

**Comment by fly-studio** [comment]
@10000go I wrote a new sync tool by myself, 
So the new tool could renames  like the variant `${SCHAME}` `${TABLE}` 
or splits the index per a date maths like `index_name-{created_at|YYYY.mm.dd}`

**Comment by 10000go** [comment]
@fly-studio is mysql-es?Has this tool been applied to online production?thx

**Comment by fly-studio** [comment]
@10000go Yes, I run this tool for 4 months, 0 data lost, of the last release.
base on alibaba/canal, zero-lost.

But the last release is not the newst version. I will releasing lately

**Comment by 10000go** [comment]
Ok, thx,I've added watch @fly-studio
